Source/WTF/wtf/StdLibExtras.h

```diff
@@ -432,6 +432,11 @@ bool findBitInWord(T word, size_t& startOrResultIndex, size_t endIndex, bool val
     return false;
 }
 
+// Used to check if a variadic list of compile time predicates are all true.
```

Comment:
I find this implementation unclear but the use is clear. Is there a coming standard for this or is this just a useful thing they are not planing to add?

Reply:
I am not sure if there is an upcoming standard, but it is used commonly in the standard library so should be. I’ll check it out and see if it has been proposed.